### PR TITLE
Peter T

### PR DESCRIPTION
**Pull request by pjten28**
testing

### CODE REVIEW

File: 03LessonThree/in-class-project/daily-planet.html [hunk 1]
```diff
@@ -1,10 +1,54 @@
...
         <link rel="stylesheet" href="css/daily-planet-style.css"/>
+
     </head>
+
     <body>
-        <!-- You code start here -->
+
+      <header>
+        <h2>The Daily Planet</h2>
+
```
Comment (bapplejax) [inline 03LessonThree/in-class-project/daily-planet.html:15]: great job here! well organized and very well spaced out to make it legible.  :dancer:

File: 04LessonFour/in-class-project/index.html [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
     <h2>Intel Core i7</h2>
```
Comment (bapplejax) [inline 04LessonFour/in-class-project/index.html:26]: good start on this, make sure you finish and sync when all done.

File: 04LessonFour/in-class-project/css/style.css [hunk 3]
```diff
@@ -21,5 +21,13 @@ h1 {
```
Comment (bapplejax) [inline 04LessonFour/in-class-project/css/style.css:22]: using what you know now about floats, try going back to this project to improve it! the spacing and alignment of your elements are still off.

File: 06LessonSix/in-class-project/css/intel-style.css [hunk 4]
```diff
@@ -8,43 +8,42 @@ header, main, footer {
     margin: auto;
 }
 
-!header {
+header {
     background-color: #0071c5;
     padding: 10px;
 }
```
Comment (bapplejax) [inline 06LessonSix/in-class-project/css/intel-style.css:14]: great job finding all the errors!

**Comment by maddiesolz** [comment]
Nice job on the Lesson 8 Homework! I will write some comments on your In-Class Project once you submit your final commit.

**Comment by maddiesolz** [comment]
Lesson 9 in class project looks pretty good! It is missing a gradient on the photos, and the second page could use some margins/padding adjustments. Also try to remember to change the <title></title> to match the site. Right now it says "Intel"

**Pull request closed without merging**